administrator/language/en-GB/com_cookiemanager.ini

```diff
@@ -9,15 +9,26 @@ COM_COOKIEMANAGER_CATEGORIES="Categories"
 COM_COOKIEMANAGER_CONFIGURATION="Cookie Manager: Options"
 COM_COOKIEMANAGER_CONSENT_BUTTON_TEXT="Cookie consent"
 COM_COOKIEMANAGER_COOKIE_DESCRIPTION="This website uses cookies to ensure that we give you the best experience on our website."
+COM_COOKIEMANAGER_COOKIE_DESC_HINT="It registers a unique ID by Google that is used to generate statistical data on how the visitor uses the website."
```

Comment:
Is that text really true for all cookies? To me it seems to be related to Google Analytics only.

Reply:
it's only a hint that is related to specific cookie `_gid`

Reply:
But as I said it refers to Google. Does it for all cookies have something to do with Google? If not, than make a more common test not referring to Google.

Reply:
then will it be ok to remove Google i think

Reply:
Replace "Google" by "provider" maybe, so it is "It registers a unique ID by the provider", or whatever else is suitable.

Reply:
Ok